app/styles/app.scss

```diff
@@ -581,7 +581,7 @@ p a:hover {
         width: 100%;
 
         .ember-version-graphic {
-          $height: 132px;
+          $height: 161px;
```

Comment:
This makes the ember-version-graphic a tad bit larger, so 3 version numbers could fit per column. I also think it looks a bit better with it larger